.github/workflows/comment_trigger_test_itensortdvp.yml

```diff
@@ -51,6 +51,8 @@ jobs:
         run: |
           using Pkg;
           Pkg.add("ITensorTDVP");
+          Pkg.activate(temp=true);
```

Comment:
I think you should activate the project before adding any packages (i.e. before the line `Pkg.add("ITensorTDVP");`).